stub.c

```diff
@@ -14,18 +14,30 @@
 
 DECLARE_SBAT(SBAT_STUB_SECTION_TEXT);
 
-static void parse_cmdline(char16_t *p) {
+static bool parse_string(char16_t *p, const char16_t *opt) {
+        const size_t opt_len = strlen16(opt);
+        if (strncmp16(p, opt, opt_len) == 0 &&
+                        (p[opt_len] == ' ' ||
+                         p[opt_len] == '\0'))
+                return true;
+        return false;
+}
 
+static void parse_cmdline(char16_t *p) {
         assert(p);
-
         while (*p != '\0') {
-                const char16_t *debug = L"debug";
-                size_t debug_len = strlen16(debug);
-                if (strncmp16(p, debug, debug_len) == 0 &&
-                                (p[debug_len] == ' ' ||
-                                 p[debug_len] == '\0'))
+                const size_t debug_len = strlen16(L"debug");
```

Comment:
Please, be consistent and look for `stubble.debug=true` or simply `stubble.debug`. Why use a variable debug_len here and do it differently for `dtb_override`. I am fine without extra variable.

Reply:
The reason I use `debug` is because that is a valid linux kernel option and I thought it would make sense to also enable debug in stubble if that is being passed. I suppose we could add an optional `stubble.debug` too?

The variable here is simply because it is being used 3 times vs 1, but I can remove that